src/OpenTelemetry.Instrumentation.Http/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Fixed an issue with `Filter` and `Enrich` callbacks not firing under certain
+  conditions when gRPC is used
+  ([#2698](https://github.com/open-telemetry/opentelemetry-dotnet/issues/2698))
```

Comment:
Should be:
([#2698](https://github.com/open-telemetry/opentelemetry-dotnet/pull/2698))

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/2750